src/sagemaker/remote_function/client.py

```diff
@@ -82,49 +82,174 @@ def remote(
-            containers is encrypted for the training job. (default: ``False``).
+          remote function. Only one of ``pre_execution_commands`` or ``pre_execution_script``
+          can be specified at the same time. Defaults to None.
+
+        environment_variables (Dict): The environment variables used inside the decorator function.
+          Defaults to ``None``.
+
+        image_uri (str): The universal resource identifier (URI) location of a Docker image on
+          Amazon Elastic Container Registry (ECR). Defaults to the following based on where the SDK
+          is running:
+
+            * For SageMaker Studio notebook cases, the image used as the kernel image for the
+              notebook is used.
+            * For other cases, it is resolved to base python image with the same python version
+              as the environment running the local code.
+
+          If no compatible image is found, a ValueError is thrown.
+
+        include_local_workdir (bool): A flag to indicate that the remote function should include
+          local directories. Set to ``True`` if the remote function code imports local modules and
+          methods that are not available via PyPI or conda. Default value is ``False``.
+
+        instance_count (int): The number of instance to use. Defaults to 1.
+
+        instance_type (str): The Amazon Elastic Compute Cloud (EC2) instance type to use to run
+          the SageMaker job. Valid values include ml.c4.xlarge. If not provided, ValueError is
+          thrown.
+
+        job_conda_env (str): The name of the conda environment to activate during job's runtime.
+          Defaults to ``None``.
+
+        job_name_prefix (str): The prefix used used to create the underlying SageMaker job.
+
+        keep_alive_period_in_seconds (int): The duration in seconds to retain and reuse provisioned
+          infrastructure after the completion of a training job, also known as SageMaker managed
+          warm pools. The use of warmpools reduces the latency time spent to provision new
+          resources. The default value for ``keep_alive_period_in_seconds`` is 0.
+          NOTE: Additional charges associated with warm pools may apply. Using this parameter will
+          also activate a new persistent cache feature, which will further reduce job start up
+          latency than over using SageMaker managed warm pools alone by caching the package source
+          downloaded in the previous runs.
+
+        max_retry_attempts (int): The max number of times the job is retried after an on
+          ```InternalServerFailure``` Error. Defaults to 1.
+
+        max_runtime_in_seconds (int): The upper limit in seconds to be used for training. After
+          this specified amount of time, SageMaker terminates the job regardless of its current
+          status. The max_run_duration time defaults to 1 day or (86400 seconds).
```

Comment:
Should this be `The max_runtime_in_seconds time defaults to 1 day or (86400 seconds).`

Reply:
fixed